### ISSUE TITLE
'position' payload for playerdig and place gives wrong coordinates

### ISSUE THREAD

**Issue by suresttexas00** [benbaptist/minecraft-wrapper#165]
```
[14:03:05] [Wrapper.py/INFO] SurestTexas00 executed: /list
[14:03:22] [Wrapper.py/INFO] SurestTexas00 executed: /say i will click on -9842
63 -8863
[14:03:22] [Server thread/INFO]: [SurestTexas00] i will click on -9842 63 -8863
{'player': <api.player.Player instance at 0x0449CFD0>, 'item': {'count': 1, 'id'
: 271, 'damage': 0}, 'position': (-9842L, 63L, -8863L)}
[14:03:37] [Wrapper.py/INFO] SurestTexas00 executed: /say i will now click on -9
842 63 -8864
[14:03:37] [Server thread/INFO]: [SurestTexas00] i will now click on -9842 63 -8
864
{'player': <api.player.Player instance at 0x0449CFD0>, 'item': {'count': 1, 'id'
: 271, 'damage': 0}, 'position': (-9842L, 63L, 67100000L)}
```

**Comment by suresttexas00** [comment]
I am seeing this in the packets too:
(0x23)

```
[23:56:58] [Wrapper.py/INFO] SurestTexas00 executed: /tp -9842 63 -8864
[23:56:58] [Server thread/INFO]: [SurestTexas00: Teleported SurestTexas00 to -98
41.5, 63.0, -8863.5]
{'location': (-9890L, 80L, 67099963L), 'id': 48}
{'location': (-9948L, 105L, 67099992L), 'id': 32}
{'location': (-9953L, 103L, 67099991L), 'id': 48}
{'location': (-9967L, 109L, 67099940L), 'id': 48}
{'location': (-9918L, 76L, -8849L), 'id': 48}
{'location': (-9947L, 105L, 67099993L), 'id': 32}
{'location': (-9888L, 80L, 67099965L), 'id': 32}
{'location': (-9860L, 75L, 67099877L), 'id': 32}
{'location': (-9861L, 74L, 67099879L), 'id': 32}
{'location': (-9947L, 105L, 67099993L), 'id': 48}
{'location': (-9890L, 80L, 67099963L), 'id': 32}
{'location': (-9858L, 75L, 67099879L), 'id': 32}
{'location': (-9859L, 75L, 67099878L), 'id': 32}
{'location': (-9919L, 81L, 67099981L), 'id': 32}
{'location': (-9982L, 100L, -8844L), 'id': 48}
{'location': (-9982L, 100L, -8844L), 'id': 32}
{'location': (-9912L, 44L, -8777L), 'id': 0}
{'location': (-9888L, 80L, 67099963L), 'id': 32}
{'location': (-9942L, 80L, -8859L), 'id': 48}
{'location': (-9890L, 80L, 67099963L), 'id': 48}
```

**Comment by suresttexas00** [comment]
I have two functions using this that behave quite differently as a result of this:
1) A /fill command using the ax "wand" - gets quite upset when it try to fill outside of 29M blocks
2) A region guard that protects player builds-  has no issues with the "different" numbering system, since it compares the playerdig 68M position to the 68M position that was used to define the region :P

**Comment by suresttexas00** [comment]
fixed by #173
